pkgs/applications/blockchains/lightning-terminal/default.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs ? import <nixpkgs> { system = builtins.currentSystem; },
+  lib ? pkgs.lib,
+  fetchFromGitHub ? pkgs.fetchFromGitHub,
+  fetchYarnDeps ? pkgs.fetchYarnDeps,
+  nodejs ? pkgs.nodejs,
+  yarn ? pkgs.yarn,
+  gnumake ? pkgs.gnumake,
+  go ? pkgs.go,
+  git ? pkgs.git,
+  stdenv ? pkgs.stdenv,
+  mkYarnPackage ? pkgs.mkYarnPackage,
+  fixup_yarn_lock ? pkgs.fixup_yarn_lock,
+  buildGoModule ? pkgs.buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "lightning-terminal";
+  version = "0.12.4-alpha";
+  src = fetchFromGitHub {
+    owner = "lightninglabs";
+    repo = "lightning-terminal";
+    rev = "v${version}";
+    hash = "sha256-gGo9abd12zZPPTBkmEfB8BNK1ZVpea2uZNpeYRb5l00=";
+  };
+
+  app = stdenv.mkDerivation {
```

Comment:
This may be a good fit for `mkYarnPackage` - depends on how the go program packages the frontend 

You could do something like
```
let
  frontend = mkYarnPackage { build it here }
in
buildGoModule...```